tests/cypress/integration/landingPage.spec.js

```diff
@@ -44,11 +46,11 @@ describe('App', () => {
       );
   });
 
-  it('displays report a repair text on the landing page', () => {
+  it('displays report a repair text on the landing page', () => {    
```

Comment:
This test is testing multiple things and depends on `enableLeaseholderFlow()` - it needs to be rewritten as two different tests each testing for specific values.